hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/enumerator/HoodieEnumeratorPosition.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.source.enumerator;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/***
+ * The position of {@link HoodieContinuousSplitEnumerator}.
+ */
+public class HoodieEnumeratorPosition implements Serializable {
+  private final String lastInstant;
```

Comment:
`lastInstant` maybe not needed now, because we now using completion time based incremental reading, and `lastInstant` in `StreamReadMonitoringFunction` is only used for backward compatibility for flink state, which should be a problem for the new flip-27 source.

Reply:
we can keep it for logging purposes.